snafu/benchmarks/uperf.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/env python3
```

Comment:
i think we probably want to have a directory called `uperf` here instead of putting the file directly under `benchmarks`. Each benchmark will probably have it's own tests, dockerfile, and potentially config file.

Something like

benchmarks/
   uperf/
      wrapper.py
      config.yaml
      Dockerfile

Reply:
I was thinking the the same, we can also consider creating subpackages within the benchmarks module

Reply:
agreed... We used to house all the other non-code stuff outside of `snafu/benchmark-wrapper` but that made the build of the container images more complex... I prefer we keep it together, but I am open to ideas if we want to break it out again...

Reply:
I popped uperf back into a directory, using the original structure. I originally made the change, as I personally am a fan of keeping code and non-code separate. Maybe we can discuss this offline some more in the future and revisit our build process.